mne/utils.py

```diff
@@ -1172,8 +1172,8 @@ def run_subprocess(command, verbose=None, *args, **kwargs):
     *args, **kwargs : arguments
         Additional arguments to pass to subprocess.Popen.
 
-    Returns
     -------
+    Returns
```

Comment:
Ha, this isn't ready for review yet, first I have to see if I can convince @agramfort to allow all of the clumsy magic for the sake of a convenience plotting function.